module-2/backend/week_two.md

```diff
@@ -13,9 +16,19 @@ end
 ```
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
+.all
+.find()
+.where()
+.first
+.last
+
+You have access to them because the Team class is inheriting from the ActiveRecord::Base class.
 
 3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
 
+Team.find(4).name
+Team.where(owner_id: 4)
```

Comment:
This will find the Team that has an owner who has an id of 4. We want to find the **owner**.

```ruby
owner_id = Team.find(4).owner_id
Owner.find(owner_id)
```

Reply:
@s-espinosa So I'll be honest, I knew it would return a team with the owner of an id of 4, but the way I understood the question was under the assumption there was only a Team model and not an Owner model. I think the last line of question 3 made me make that assumption.

But either way, good to know.